### PR TITLE
2567: working copy case filtering in computed

### PR DESCRIPTION
**Pull request by rachaelparris**

### CODE REVIEW

File: web-client/src/presenter/actions/TrialSessionWorkingCopy/computeTrialSessionWorkingCopyFilterValuesAction.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      settled: false,
+      showAll: false,
+      statusUnassigned: false,
+      takenUnderAdvisement: false,
+    });
+  } else if (props.key.includes('filters') && props.value === false) {
+    store.set(state.trialSessionWorkingCopy.filters.showAll, false);
+  } else if (
+    props.key.includes('filters') &&
+    filters.aBasisReached === true &&
```
Comment (kkoskelin) [inline web-client/src/presenter/actions/TrialSessionWorkingCopy/computeTrialSessionWorkingCopyFilterValuesAction.js]: these can probably be shortened to `filters.aBasisReached && filters.continued && ....`?

File: web-client/src/presenter/computeds/trialSessionWorkingCopyHelper.js [hunk 2]
```diff
@@ -15,7 +15,8 @@ const compareCasesByPractitioner = (a, b) => {
 export const trialSessionWorkingCopyHelper = (get, applicationContext) => {
   const { TRIAL_STATUS_TYPES } = get(state.constants);
   const trialSession = get(state.trialSession) || {};
-  const { sort, sortOrder } = get(state.trialSessionWorkingCopy) || {};
+  const { caseMetadata, filters, sort, sortOrder } =
+    get(state.trialSessionWorkingCopy) || {};
```
Comment (kkoskelin) [inline web-client/src/presenter/computeds/trialSessionWorkingCopyHelper.js]: In the case this defaults to `{}` is the code safe to execute or  will there be problems if `filters` or `caseMetadata` turn out to be null?

**Review by kkoskelin — CHANGES_REQUESTED** [review]
Might be one thing to address in case some undefined values popped up.

File: web-client/src/presenter/computeds/trialSessionWorkingCopyHelper.js (hunk 2 above)
Comment (rachaelparris) [reply web-client/src/presenter/computeds/trialSessionWorkingCopyHelper.js]: Addressed!

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**